examples/color_palette/src/main.rs

```diff
@@ -284,19 +284,25 @@ impl<C: 'static + ColorSpace + Copy> ColorPicker<C> {
         let [s1, s2, s3] = &mut self.sliders;
         let [cr1, cr2, cr3] = C::COMPONENT_RANGES;
 
+        fn slider<C>(
+            state: &mut slider::State,
+            range: RangeInclusive<f64>,
+            component: f32,
+            update: impl Fn(f32) -> C + 'static,
+        ) -> Slider<f64, C> {
+            Slider::new(state, range, f64::from(component), move |v| {
+                update(v as f32)
+            })
+            .step(0.01)
+        }
+
```

Comment:
I assume that the point here is to have a slider with higher precision, even though the color components can only have `f32` values?

Reply:
Yes, that's correct. This allows us to use a `0.01` step without precision issues.